bedrock/firefox/views.py

```diff
@@ -396,9 +396,6 @@ def get_template_names(self):
             else:
                 template = 'firefox/dev-firstrun.html'
         elif show_62_firstrun(version):
-            if locale == 'en-US' and experience == 'firefox-election':
```

Comment:
Looks like there are some flake8 errors:

```
bedrock/firefox/views.py:390:1: F841 local variable 'experience' is assigned to but never used
bedrock/firefox/views.py:391:1: F841 local variable 'locale' is assigned to but never used
```

Reply:
@alexgibson -thanks Alex! Should be cleaned up now. 👍